### DISCUSSION TITLE
Slow page loading

### DISCUSSION THREAD

**Discussion by Abbasjafri-syed** [solodit/feedback#28] [Bug Report]
I usually face problem of slow page loading....as site keeps on loading
![image](https://github.com/solodit/feedback/assets/101616115/ecc60a24-d581-41cc-b276-2a5e139e6ed6)
![image](https://github.com/solodit/feedback/assets/101616115/ccaef07d-d37e-4b90-8532-17369661a2fa)

**Comment by hans-cyfrin** [comment]
We are working on the optimization this week. Please stay tuned.

**Comment by hans-cyfrin** [comment]
I believe there have been some improvements here. Will close for now.
